src/contracts/pods/EigenPod.sol

```diff
@@ -201,19 +223,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         BeaconChainProofs.verifyValidatorFields(
             validatorIndex,
             beaconStateRoot,
-            proofs.validatorFieldsProof,
+            proof,
             validatorFields
         );
 
-        // verify ETH validator's current balance, which is stored in the `balances` container of the beacon state
-        BeaconChainProofs.verifyValidatorBalance(
-            validatorIndex,
-            beaconStateRoot,
-            proofs.validatorBalanceProof,
-            proofs.balanceRoot
-        );
         // set the status to active
-        validatorStatus[validatorIndex] = VALIDATOR_STATUS.ACTIVE;
+        _validatorPubkeyHashToInfo[validatorPubkeyHash].status = VALIDATOR_STATUS.ACTIVE;
```

Comment:
prob good to find a way to batch the `_validatorPubkeyHashToInfo` SSTORES into one. maybe create the struct in mem and store it at the end at once